src/_backend_agg_wrapper.cpp

```diff
@@ -333,12 +333,13 @@ PyRendererAgg_draw_path_collection(PyRendererAgg *self, PyObject *args, PyObject
     PyObject *offset_position; // offset position is no longer used
 
     if (!PyArg_ParseTuple(args,
-                          "O&O&OO&O&O&O&O&O&O&O&OO:draw_path_collection",
+                          "O&O&O&O&O&O&O&O&O&O&O&OO:draw_path_collection",
```

Comment:
I don't know what these strings are.  Was this just a typo before?

Reply:
No, this needs to be changed because I added `&convert_pathgen`.  See https://docs.python.org/3/extending/extending.html#extracting-parameters-in-extension-functions